docker/Dockerfile.local

```diff
@@ -0,0 +1,23 @@
+from espressopp/buildenv:ubuntu
+
+ARG EXTERNAL
+ARG CC
+ARG CXX
+ARG BUILD_UG
+
+COPY . /home/espressopp/espressopp
+USER root
+#RUN chown -R espressopp:espressopp /home/espressopp/espressopp
+#USER espressopp
+
+WORKDIR /home/espressopp/espressopp
+RUN mkdir build
+
+WORKDIR build
+RUN pip install h5py
```

Comment:
We could add `h5py` to `buildenv`!

Reply:
if we add this it will invoke a new series of tests making the time required for travis checks 2 times longer. Is it right?